action/protocol/staking/read_state.go

```diff
@@ -123,3 +137,30 @@ func getPageOfBuckets(buckets []*VoteBucket, offset, limit int) []*VoteBucket {
 	}
 	return res
 }
+
+func toIoTexTypesCandidateListV2(candidates CandidateList) *iotextypes.CandidateListV2 {
+	res := &iotextypes.CandidateListV2{
+		Candidates: make([]*iotextypes.CandidateV2, 0, len(candidates)),
+	}
+	for _, c := range candidates {
+		typCand := c.toIoTexTypes()
+		res.Candidates = append(res.Candidates, typCand)
+	}
+	return res
```

Comment:
nit: define res := iotextypes.CandidateListV2{
and return &res?

once saw one guide says this clearly indicates returning a pointer/address

Reply:
i think it doesn't matter since function signature indicate it returns pointer